Sources/swift-bootstrap/main.swift

```diff
@@ -116,12 +116,12 @@ struct SwiftBootstrapBuildTool: AsyncParsableCommand {
     @Flag()
     public var useIntegratedSwiftDriver: Bool = false
 
-    /// A flag that indicates this build should check whether targets only import
+    /// An option that indicates this build should check whether targets only import
     /// their explicitly-declared dependencies
-    @Option()
+    @Option(help: "An option that indicates this build should check whether targets only import their explicitly-declared dependencies")
```

Comment:
nit: IMO it's a good opportunity to make the help entry more readable.

```suggestion
    @Option(help: "Check that targets only import their explicitly-declared dependencies")
```